modules/rosa-required-aws-service-quotas.adoc

```diff
@@ -124,16 +158,20 @@ The default value of 5 vCPUs is not sufficient to create ROSA clusters. ROSA has
 |50
 |The maximum number of Application Load Balancers that can exist in each region.
 
+ifndef::openshift-rosa-hcp[]
 |Classic Load Balancers per Region
 |elasticloadbalancing
 |L-E9E9831D
 |20
 |20
 |The maximum number of Classic Load Balancers that can exist in each region.
+endif::openshift-rosa-hcp[]
+
 |===
 
 [role="_additional-resources"]
 == Additional resources
 * link:https://aws.amazon.com/premiumsupport/knowledge-center/request-service-quota-increase-cli/[How can I request, view, and manage service quota increase requests using AWS CLI commands?]
 * link:https://docs.aws.amazon.com/ROSA/latest/userguide/service-quotas-rosa.html[ROSA service quotas]
-* link:https://docs.aws.amazon.com/servicequotas/latest/userguide/request-quota-increase.html[Request a quota increase]
\ No newline at end of file
+* link:https://docs.aws.amazon.com/servicequotas/latest/userguide/request-quota-increase.html[Request a quota increase]
+* link:https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_iam-quotas.html[AWS documentation: IAM and AWS STS quotas]
```

Comment:
```suggestion
* link:https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_iam-quotas.html[IAM and AWS STS quotas (AWS documentation)]
```

https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#lead-in-link-sentence-style-for-the-link-attribute